modules/cudf/test/cudf-series-test.ts

```diff
@@ -469,3 +480,68 @@ ${[true, null, true]} | ${[true, true, true]}
 
   expect([...result]).toEqual(expected);
 });
+
+test('Series.TimestampDay', () => {
+  const dateTime = Math.floor(new Date(2021, 4, 13, 20, 11, 30, 100).getTime() / 86400000);
+  const s        = Series.new({type: new TimestampDay, data: new Int32Buffer([dateTime])});
```

Comment:
not sure how to make these tests work without specifying `new Int32Buffer` in the `data` key here :thinking:

Reply:
1. Add another [`Series.new()` overload](https://github.com/rapidsai/node-rapids/blob/781d9f12465ce0d570e14e91cf5f67794b430f3b/modules/cudf/src/series.ts#L231-L242) for `input: (Date|null|undefined)[]): Series<TimestampMillisecond>`
  a. Don't forget to add one for lists of Dates! ([example](https://github.com/rapidsai/node-rapids/blob/781d9f12465ce0d570e14e91cf5f67794b430f3b/modules/cudf/src/series.ts#L285-L298))
2. Check for JS Date in the [`inferType()` function](https://github.com/rapidsai/node-rapids/blob/781d9f12465ce0d570e14e91cf5f67794b430f3b/modules/cudf/src/series.ts#L916-L983) and return a `TimestampMillisecond` instance.